Test/dafny0/Termination.dfy

```diff
@@ -487,7 +487,7 @@ datatype Tree = Empty | Node(root: int, left: Tree, right: Tree)
       }
   }
 
-  lemma {:induction this} EvensSumToEven()  // explicitly use "this" as quantified over by induction hypothesis
+  lemma {:induction this} {:vcs_split_on_every_assert} EvensSumToEven()  // explicitly use "this" as quantified over by induction hypothesis
```

Comment:
I discovered these during the creation of this branch, but it couldn't hurt to include them in 3.x, as well.